tests/test_client.py

```diff
@@ -39,7 +39,9 @@ def test_get_channels_status(app):
     }
 
 
-def test_post_message(app):
+@pytest.mark.parametrize("test_message", ["hello world", "test message", ":tada:"])
+@pytest.mark.parametrize("test_target_id", ["CXXXX", "TXXXX", "CSD2310124", "C23231"])
```

Comment:
Lets use this to also check the cases for incorrect paramas and error test, or separate them into different test functions

Reply:
Ping on this comment

Reply:
I gave a shot, is this what you were expecting?